force-app/main/apex/examples/integration/WDController.cls

```diff
@@ -0,0 +1,101 @@
+/**
+ * Returns data from a Workday Reporting as a Service report (RaaS)
+ * It requires that you setup a Named Credential in Salesforce Setup
+ * so you don't have to show the login/pw credentials in the code
+ * 
+ *
+ * @author geoffrothman
+ */
+@RestResource(urlMapping='/wdraas')
+global class WDController {
+    
+    @HttpPost   
+    global static String wdraas() {
+        
+        // fetch data
+        HttpRequest request = new HttpRequest();
+        Http http = new Http();
+        // make sure this domain is whitelisted in the proxy
+        //request.setEndpoint('https://wd2-impl-services1.workday.com/ccx/service/customreport2/salesforce9/sfdc_integration_user1/Einstein-Job-Catalog?format=json');
+        request.setEndpoint('callout:WorkdayRaaS_Named_Credential');
```

Comment:
Can you integrate the `ApexStepRequest` to take the parts of the workday request as parameters that can be passed in via dashboard step query body?